tests/bearlib/aspects/BaseTest.py

```diff
@@ -15,5 +15,7 @@ def test_class(self):
         assert not isinstance(aspectbase, aspectclass)
 
     def test_init_needs_aspectclass(self):
-        with pytest.raises(AttributeError):
+        error_str = 'object has no attribute'
+        with pytest.raises(AttributeError) as exc:
```

Comment:
@userzimmermann deprecated the use of unittest and since pytest is the test runner and is also backwards compatible. So, IMHO, it would be good to reconsider as there won't be any visible performance improvements, neither would it make code any more readable.
I'ld highly respect your decision @jayvdb